cross-compile-riscv.sh

```diff
@@ -0,0 +1,33 @@
+#/bin/bash
+
+# Cross-compile for RISC-V in an Ubuntu 22.04 container
```

Comment:
Probably this whole file should go into a new section in the Dockerfile.dapper? I'm not entirely sure but that feels more correct given our build pattern in this repo. We don't currently support cross-compiling so I admit there's not a great place to fit this in as-is.

Reply:
This script is deprecated, so I have removed it. Instead of creating a new build environment in Ubuntu, I now install a cross-compiler and dependencies in the existing build container.